src/DotNet.Testcontainers/Containers/TestcontainersContainer.cs

```diff
@@ -181,6 +182,15 @@ public Task CopyFileAsync(string filePath, byte[] fileContent, int accessMode =
       return this.client.CopyFileAsync(this.Id, filePath, fileContent, accessMode, userId, groupId, ct);
     }
 
+    public Task ExportFilesFromContainerAsync(string containerPath, Stream outStream)
```

Comment:
This should be part of the interface `IRunningDockerContainer` incl. documentation, etc. I suggest a different name too, relative to `CopyFileAsync`. Maybe `ReadFileAsync` or `GetFileAsync`.